apps/server/src/shared/infra/rabbitmq/exchange/files-storage.ts

```diff
@@ -0,0 +1,41 @@
+import { EntityId, FileRecordParentType, ScanStatus } from '@shared/domain';
```

Comment:
The FileRecordParentType why not define it on this place and use in file file-storage-client? The mapper already exists.